### ISSUE TITLE
Consensus changes list becomes stale

### ISSUE THREAD

**Issue by sumiflow** [decred/decrediton#3683]
On the Governance page, the "Proposals" tab automatically refreshes but the "Consensus Changes" tab does not.

When I opened Decrediton today, the new agendas were not showing.
I then closed Decrediton and restarted it and then they showed up.
It would be nice if the Consensus Changes page refreshed on its own without requiring a restart.

**Comment by alexlyp** [comment]
Well there are a few things going on behind the scenes that don't really allow for updating the list on the fly.  First the list itself is first gotten from dcrwallet, and that will never change from session to session unless you have recently updated.  The other is that is checks against dcrdata for live agendas.  That information should be static. 

I'm not sure why you saw the old ones, but updating isn't really feasible or worth doing much of a deep dive imo. NACK

**Comment by sumiflow** [comment]
I get requiring an upgrade for the list from dcrwallet to see them but this is more about the agendas still being out of date after the upgrade.  I saw the old ones because, after I updated, the new agendas weren't there on dcrdata yet.   They were added to dcrdata later, after I had already upgraded and opened Decrediton.

Just wanted to explain but I won't bother about it further if you prefer to keep it closed.  Thanks for the response.
